features/fedcm.yml

```
@@ -0,0 +1,8 @@
+name: Federated credential management
+description: The `IdentityCredential` API delegates authentication to a third-party identity provider directly, instead of depending on third-party cookies. Also known as FedCM.
```

@nsatragno before we land this, can you also help check if this is a reasonable description of FedCM? Not sure if you checked it already.

---

Let's get rid of it. It was meant to contrast with the "depending on" bit, but I can see why it would be an iffy choice. We don't lose anything in this form, I think:

```suggestion
description: The `IdentityCredential` API delegates authentication to a third-party identity provider, instead of using third-party cookies. Also known as FedCM.
```